### PR TITLE
Add Pagerduty

### PR DESCRIPTION
**Pull request by adrianteri**

### CODE REVIEW

**Comment by adrianteri** [comment]
Anything needed from my end? This site is currently ranked at 16,610 on [Alexa](https://www.alexa.com/siteinfo/pagerduty.com)

File: _data/developer.yml [hunk 1]
```diff
@@ -403,6 +403,12 @@ websites:
       - totp
     doc: https://github.com/composer/packagist/pull/1031
 
+  - name: Pagerduty
+    url: https://pagerduty.com
+    img: pagerduty.png
+    twitter: Pagerduty
```
Comment (phallobst) [inline _data/developer.yml]:
```suggestion
    twitter: pagerduty
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by adrianteri** [comment]
Super! Added the changes ...Got confused if it was the twitter @ or the account name @phallobst

**Review by phallobst — APPROVED** [review]

File: _data/developer.yml [hunk 2]
```diff
@@ -403,6 +403,12 @@ websites:
       - totp
     doc: https://github.com/composer/packagist/pull/1031
 
+  - name: Pagerduty
+    url: https://pagerduty.com
+    img: pagerduty.png
+    twitter: pagerduty
```
Comment (Carlgo11) [inline _data/developer.yml]: @PagerDutyHelp is actually better suited for this 😅 
```suggestion
    twitter: PagerDutyHelp
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/developer.yml (hunk 2 above)
Comment (adrianteri) [reply _data/developer.yml]: Resolved from pagerduty to PagerDutyHelp ...Can I get an approval on this PR @Carlgo11?

**Review by adrianteri — COMMENTED** [review]
Resolved!

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**